specification/servicenetworking/resource-manager/readme.md

```diff
@@ -78,7 +90,6 @@ This is not used by Autorest itself.
 
 ```yaml $(swagger-to-sdk)
 swagger-to-sdk:
-  - repo: azure-sdk-for-python
```

Comment:
Python SDK could be generated from Typespec so there is no need to keep configuration for swagger.

Reply:
Thanks for reviewing @msyyc , can you please also add the BreakingChange-JavaScript-Sdk-Approved label to help us complete this PR? Thanks

Reply:
Add JS Owner @kazrael2119 for JS SDK review.

Reply:
@msyyc  have we decided to release from typespec for this service?

Reply:
Python doesn't see any blocker and I don't want pipeline to generate SDK twice from swagger and typespec which is waste of pipeline resources.